templates/index.html

```diff
@@ -374,7 +369,7 @@ <h2>Leaning on a strong cloud ecosystem</h2>
             attrs={"class": "p-pull-quote__image"},
           ) | safe
         }}
-        <p class="p-pull-quote__quote">The combination of Ampere’s Arm-based servers with a provisioned virtualisation solution like Canonical’s Anbox Cloud delivers the flexible, high-performance and secure infrastructure that developers need in order to deliver a better user experience for consumers.</p>
```

Comment:
It's a bad practice to use such special characters because they might be rendered incorrectly on some browsers/locales. Also, it's wrong to have a mix of different apostrophes in different parts of the page. In some places there is this one, in some others the regular '. And this seems quite random. Let's stick to one, the most compatible one, and use it everywhere. The reason why that one is there in the copydoc is probably some automated formatting put in place by the editor. I don't think there is a deliberate choice to use it in place of the standard '.